specification/network/resource-manager/Microsoft.Network/stable/2018-04-01/virtualNetworkGateway.json

```diff
@@ -711,98 +709,6 @@
         "x-ms-long-running-operation": true
       }
     },
-    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/virtualNetworkGateways/{virtualNetworkGatewayName}/setvpnclientipsecparameters": {
-      "post": {
-        "tags": [
-          "VirtualNetworkGateways"
-        ],
-        "operationId": "VirtualNetworkGateways_SetVpnclientIpsecParameters",
```

Comment:
Can you confirm whether this API has been removed from the April version? At this point we should really be adding additive changes only

Reply:
Yes, we had this discussion during approval. that's a private feature was accidently published earlier (and removed).